pkg/sql/create_index.go

```diff
@@ -34,14 +35,27 @@ type createIndexNode struct {
 	tableDesc *sqlbase.MutableTableDescriptor
 }
 
-var indexStorageParams = map[string]storageParamType{
-	`fillfactor`:                        storageParamInt,
-	`vacuum_cleanup_index_scale_factor`: storageParamUnimplemented,
-	`buffering`:                         storageParamUnimplemented,
-	`fastupdate`:                        storageParamUnimplemented,
-	`gin_pending_list_limit`:            storageParamUnimplemented,
-	`pages_per_range`:                   storageParamUnimplemented,
-	`autosummarize`:                     storageParamUnimplemented,
+type indexStorageParamObserver struct {
+	indexDesc *descpb.IndexDescriptor
```

Comment:
why does this need an indexDesc?

Reply:
in a future state, you may alter the index descriptor based on the storage params.
(i will need to do this later, so...)

Reply:
I think that you should just add them once you need them

Reply:
ok